samples/certs/Makefile

```diff
@@ -121,3 +133,52 @@ root-key.pem:
 	@echo "O = $(CITADEL_ORG)" >> $@
 	@echo "CN = $(CITADEL_CN)" >> $@
 	@echo "L = $(L:/=)" >> $@
+
+#------------------------------------------------------------------------
+##WORKLOAD_CERTS=1 <name>-certs: generate workload certificates using above intermediate ca Includes all PEM needed.
+
+%/workload-cert-chain.pem: %/workload-cert.pem %/ca-cert.pem root-cert.pem
+	@echo "generating $@"
+	@cat $^ > $@
+
+%/workload-cert.pem: %/workload.csr
+	@echo "generating $@"
+	@openssl x509 -req -days $(WORKLOAD_DAYS) \
+		-CA $(dir $<)/ca-cert.pem  -CAkey $(dir $<)/ca-key.pem -set_serial $(CITADEL_SERIAL) \
```

Comment:
same comment with $(?D)

Here is the makefile automatic variables cheat sheet: https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html